content/posts/2020-10-29-loadsmart-take-home-exercises/index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Loadsmart Take Home Exercises
+author: Flavio S. Truzzi
+twitter: flaviotruzzi
+layout: post
+lang: en
+path: /blog/loadsmart-take-home-exercises
+date: '2020-10-29'
+comments: true
+---
+
+Hiring is something taken very seriously at Loadsmart. One of our principles states that **Our biggest asset is our People**, we invest a lot of time in our hiring process. Our process for engineering includes what we call a Take-Home Exercise, and today I would like to give some tips for doing well on these.
+
+Our exercises are tailored for different positions, so we have different exercises for Backends, Frontends, SREs, and Data Scientists. Most of the tips can be applied to all of them.
+
+### Before starting the exercise
+
+Carefully read the file we send. There we have the goal and the expectations. A lot of times, we receive exercises that do not achieve the requirements specified in the text. Focus on what we ask, this will change according to the task, but generally, we will look into:
```

Comment:
```suggestion
Carefully read the file we send. There we have the goal and the expectations. Most of the time, we receive exercises that do not achieve the requirements specified in the text. Focus on what we ask, this will change according to the task, but generally, we will look into:
```

Reply:
Idk if we should say it is most of the time. I am concerned that might sign to people that wasn't approved before that it was the main reason.